examples/cactus-example-electricity-trade/BusinessLogicElectricityTrade.ts

```diff
@@ -20,10 +20,10 @@ import {
 } from "@hyperledger/cactus-cmd-socketio-server";
 import { makeRawTransaction } from "./TransactionEthereum";
 
-const fs = require("fs");
-const yaml = require("js-yaml");
+// const fs = require("fs");
+// const yaml = require("js-yaml");
```

Comment:
probably you can simply remove these lines?

Reply:
fixed